### ISSUE TITLE
Popups doesn't work

### ISSUE THREAD

**Issue by petoblat** [thomasloven/hass-browser_mod#327]
My Home Assistant version: 0.XX.X

**What I am doing:**

**What I expected to happen:**

**What happened instead:**

**Minimal** steps to reproduce:

```yaml
# The least amount of code or steps possible to reproduce my error

# Put your code/steps here
```

**Error messages from the browser console:**

```
// Select everything from the browser console and copy it

// Paste it here
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [ ] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [ ] Have made sure I am using the latest version of the plugin.

- [ ] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [ ] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by maxxer** [comment]
Are you in HTTPS? It started working to me when I enabled secure connection

**Comment by Xeyame** [comment]
`deviceID: this` was changed.

Working example configuration of a card for me:
```
type: entities
entities:
  - entity: light.outside
    tap_action:
      action: fire-dom-event
      browser_mod:
        command: popup
        title: Outside lights
        card:
          type: entities
          entities:
            - entity: light.outside1
            - entity: light.outside2
            - entity: light.outside3
hold_action:
  action: none
```

**Comment by kaizersoje** [comment]
Popups don't seem to be working on my Android phone since upgrading to 2022.6.x

It seems to be working fine on my laptop though.

**Comment by damusmf** [comment]
running in to same issues of the pop ups not working in the android companion app.  They work on desktop and one web browser on mobile, just not in the companion app.  Similar issue posted here: https://github.com/UI-Lovelace-Minimalist/UI/issues/668

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
